test/chpldoc/linkage-spec/exportTest.doc.catfiles

```diff
@@ -1,2 +1,2 @@
-docs/exportTest.doc.txt
-docs/exportTest.doc/Other.txt
+docs/_sources/modules/exportTest.doc.txt
+docs/_sources/modules/exportTest.doc/Other.txt
```

Comment:
What determines this _sources/modules directory?  Is there a(n easy) change we could make to how chpldoc testing works that would cause it to use the old docs/ directory?

Also: I was surprised that the extension on the file was .txt rather than .rst.  Is that our choice or something else's?